src/drivers/camera_trigger/camera_trigger_params.c

```diff
@@ -136,3 +136,16 @@ PARAM_DEFINE_INT32(TRIG_PINS, 56);
  * @group Camera trigger
  */
 PARAM_DEFINE_FLOAT(TRIG_DISTANCE, 25.0f);
+
+/**
+ * Camera feedback mode
+ *
+ * Sets the camera feedback mode. This parameter will be moved once we
+ * support better camera feedback.
+ *
+ * @value 0 No camera feedback available
+ * @min 0
```

Comment:
This was left in by mistake after relocating. The real one is in the feedback module. I removed it now.